src/interactive-auth.js

```diff
@@ -117,7 +124,9 @@ function InteractiveAuth(opts) {
     this._chosenFlow = null;
     this._currentStage = null;
 
-    this._polling = false;
+    // if we are currently trying to submit an auth dict (which includes polling)
+    // the promise the will resolve/reject when it completes
+    this._submitPromise = false;
```

Comment:
Maybe `null` is more natural to indicate the state without a `Promise` value?

```suggestion
    this._submitPromise = null;
```